system/jlib/jsocket.cpp

```diff
@@ -4871,6 +4884,11 @@ class CSocketEpollThread: public CSocketBaseThread
         if (removeSock(sock))
         {
             selectvarschange = true;
+            // stop if no more fds on this thread
+            // and more than one thread is present
+            unsigned n = items.ordinality();
+            if (n == 0 && numThreads > 1)
```

Comment:
Is this worth doing? It adds quite a bit of complication.  he Select code doesn't have an equivalent.  How likely is the code to listen to lots of sockets, stop listening and not listen again?